Lecture2/task1/Vegetable.java

```diff
@@ -0,0 +1,12 @@
+abstract public class Vegetable {
+    protected int calories;
+
```

Comment:
add weight in grams for more readable logic

Reply:
added